Crashlytics/UnitTests/FIRCLSReportUploaderTests.m

```diff
@@ -114,17 +115,46 @@ - (void)testUrgentUploadPackagedReportWithPath {
   [self runUploadPackagedReportWithUrgency:YES];
 }
 
-- (void)testUploadPackagedReportWithoutDataCollectionToken {
-  [self setUpForUpload];
+- (void)testUrgentWaitUntillUpload {
```

Comment:
Curious, do these added tests deadlock without the semaphore signaling?

Reply:
Yep) tests will fail with the timeout error